pubspec.yaml

```diff
@@ -27,18 +27,25 @@ environment:
 # the latest version available on pub.dev. To see which dependencies have newer
 # versions available, run `flutter pub outdated`.
 dependencies:
+  flutter:
+    sdk: flutter
+
+  # The following adds the Cupertino Icons font to your application.
+  # Use with the CupertinoIcons class for iOS style icons.
+  cupertino_icons: ^1.0.5
   firebase_auth: ^3.6.3
   firebase_core: ^1.20.1
   google_maps_flutter: ^2.1.10
   flutter_bloc: ^8.0.1
   intl: ^0.17.0
+  shared_preferences:
   nartus_ui_package:
     path: packages/nartus_ui_package
   nartus_location:
     path: packages/nartus_location
   nartus_authentication:
     path: packages/nartus_authentication
-
+  internet_connection_checker: ^0.0.1+4
```

Comment:
Please do not use this package. There is another story in which we'll need connectivity check, and we'll use connectivity_plus, which is a Flutter Favourite package.